@xen-orchestra/lite/src/components/vm/VmHeader.vue

```diff
@@ -93,20 +136,45 @@ import {
 } from "@fortawesome/free-solid-svg-icons";
 import { computed } from "vue";
 import { useRouter } from "vue-router";
+import { computedAsync } from "@vueuse/core";
 
 const vmStore = useVmStore();
 const hostStore = useHostStore();
 const poolStore = usePoolStore();
 const { currentRoute } = useRouter();
 
-const vm = computed(() =>
-  vmStore.getRecordByUuid(currentRoute.value.params.uuid as string)
+const hasOperation = (operation: string) =>
+  Object.values(vm.value.current_operations).includes(operation);
+
+const vm = computed(
+  () => vmStore.getRecordByUuid(currentRoute.value.params.uuid as string)!
 );
+const xenApi = computedAsync(() => useXenApiStore().getXapi());
 const name = computed(() => vm.value?.name_label);
 const isRunning = computed(() => vm.value?.power_state === "Running");
 const isHalted = computed(() => vm.value?.power_state === "Halted");
 const isSuspended = computed(() => vm.value?.power_state === "Suspended");
-const isPaused = computed(() => vm.value?.power_state === "Paused");
+const isPaused = computed(
```

Comment:
IMHO, `isPaused` should only return the state.

Reply:
I think it's better to use the actual power_state of the VM. We don't know exactly why it goes through a Paused state, it might change in the future, etc. So let's not make any assumption and show whatever XAPI reports.